GetJobProfiles/ContentRecipes/SharedContent.recipe.json

```diff
@@ -107,6 +107,126 @@
                     "TitlePart": {
                         "Title": "Get help using this service"
                     }
+                },
+                {
+                    "ContentItemId": "40w8mzcnvknysz6mvdah1mse4t",
+                    "ContentItemVersionId": "46qmx4xwb0fh7vp9h624trdg4k",
+                    "ContentType": "SharedContent",
+                    "DisplayText": "Support during Covid-19",
+                    "Latest": true,
+                    "Published": true,
+                    "ModifiedUtc": "2020-07-02T08:43:01.5354219Z",
+                    "PublishedUtc": "2020-07-02T08:43:01.5499142Z",
+                    "CreatedUtc": "2020-07-02T08:43:01.5354219Z",
+                    "Owner": "StaxEditorAdmin",
```

Comment:
Other content items use `admin`. `StaxEditorAdmin` won't be available locally.
We might need to give this some more thought, but for now, we should make all the content items consistent.

Reply:
updated to be consistent